### DISCUSSION TITLE
Remove Qt dependencies

### DISCUSSION THREAD

**Discussion by microhobby** [slint-ui/slint#3485] [Q&A] [answered]
I'm writing a shared library using the Slint Rust, and the result is linking Qt6 dependencies:

```bash
        libQt6Core.so.6 => /usr/lib/Qt/6.2.4/gcc_64/lib/libQt6Core.so.6 (0x00007f21a181f000)
        libQt6Gui.so.6 => /usr/lib/Qt/6.2.4/gcc_64/lib/libQt6Gui.so.6 (0x00007f21a0ece000)
        libQt6Widgets.so.6 => /usr/lib/Qt/6.2.4/gcc_64/lib/libQt6Widgets.so.6 (0x00007f21a07aa000)
        libicui18n.so.56 => /usr/lib/Qt/6.2.4/gcc_64/lib/libicui18n.so.56 (0x00007f219fe00000)
        libicuuc.so.56 => /usr/lib/Qt/6.2.4/gcc_64/lib/libicuuc.so.56 (0x00007f219fa00000)
        libicudata.so.56 => /usr/lib/Qt/6.2.4/gcc_64/lib/libicudata.so.56 (0x00007f219e000000)
```

These are really needed? There is a way to get rid of it?

**Comment by ogoffart** [comment, marked as answer]
These are not needed.
The backend-qt feature is enabled by default. You can disable it by using default-features=false and enable the other features manually. See
https://docs.rs/slint/latest/slint/docs/cargo_features/index.html

Alternatively, you can just set the SLINT_NO_QT=1 env variable when building
